WickedEngine/wiMath.h

```diff
@@ -17,66 +17,111 @@ namespace wiMath
 	}
 	inline float Distance(const XMVECTOR& v1, const XMVECTOR& v2)
 	{
+#ifdef _WIN32
```

Comment:
Yeah, the reference doesn't really make sense here anyway, so your edit is better and should be the only one kept.